plugins/index.js

```diff
@@ -90,8 +93,13 @@ export function loadPlugin(manifest) {
         script.src = getSiteURL() + bundlePath;
         script.onload = onLoad;
         console.log('Loading ' + manifest.id + ' plugin'); //eslint-disable-line no-console
-        document.getElementsByTagName('head')[0].appendChild(script);
 
+        if (!tryRequire(bundlePath)) {
+            reject(new Error('Unable to load bundle for plugin ' + manifest.id));
+            return;
+        }
+
+        document.getElementsByTagName('head')[0].appendChild(script);
```

Comment:
Just to confirm, this doesn't re-request the network resource a second time, right? If it did, I'd worry about weird cases where /that/ fails for mysterious (but unlikely reasons).

Reply:
I've added a onError handler that should safely catch any loading/executing errors.

Reply:
Correct. I've removed the `tryRequire` check given that not only was a duplicate load of the script but was causing unexpected behavior in development environment. 

I've included the `onerror` event handling that would gracefully handle any weird cases of failure as well.